.github/workflows/e2e.ci.yaml

```diff
@@ -59,6 +59,8 @@ jobs:
       UI_REPO: ghcr.io/apache/skywalking/ui
       SATELLITE_TAG: v35bfaff6352b4dc351a706772796a1f79b651c14
       SATELLITE_REPO: ghcr.io/apache/skywalking-satellite/skywalking-satellite
+      BANYANDB_TAG: 0e734c462571dcf55dbb7761211c07d8b156521e
+      BANYANDB_REPO: ghcr.io/apache/skywalking-banyandb
```

Comment:
You are using a new BanyanDB or an older version?

Reply:
The older one that matches the OAP's client.